### PR TITLE
"textgroup entityform is not allowed" php-warnings on watchdog & drush -- 1.2.2 release

### PR DESCRIPTION
**Pull request by salvoscala**
```php
The string entityform:entityform_type:contact:draft_save_text for textgroup entityform is not allowed for translation because of its text  [warning]
format.
The string entityform:entityform_type:contact:submission_text for textgroup entityform is not allowed for translation because of its text  [warning]
format.
The string entityform:entityform_type:contact:instruction_pre for textgroup entityform is not allowed for translation because of its text  [warning]
format.
The string entityform:entityform_type:listing_enquiry:draft_save_text for textgroup entityform is not allowed for translation because of   [warning]
its text format.
The string entityform:entityform_type:listing_enquiry:instruction_pre for textgroup entityform is not allowed for translation because of   [warning]
its text format.
The string entityform:entityform_type:contact:draft_save_text for textgroup entityform is not allowed for translation because of its text  [warning]
format.
The string entityform:entityform_type:contact:submission_text for textgroup entityform is not allowed for translation because of its text  [warning]
format.
The string entityform:entityform_type:contact:instruction_pre for textgroup entityform is not allowed for translation because of its text  [warning]
format.
The string entityform:entityform_type:listing_enquiry:draft_save_text for textgroup entityform is not allowed for translation because of   [warning]
its text format.
The string entityform:entityform_type:listing_enquiry:instruction_pre for textgroup entityform is not allowed for translation because of   [warning]
its text format.
```

Reporting this as per testing of 1.2.2 release https://github.com/Roomify/RfA/releases/tag/1.2.2

### CODE REVIEW

**Comment by acrollet** [comment]
@marcrobinsone thanks for the report! We'll look into this as we're able.

File: modules/roomify/roomify_contact/roomify_contact.features.inc [hunk 1]
```diff
@@ -38,15 +38,26 @@ function roomify_contact_default_entityform_type() {
       "page_title_view" : "",
       "preview_page" : 0,
       "submission_page_title" : "",
-      "submission_text" : { "value" : "", "format" : "filtered_text" },
+      "submission_text" : { "value" : "", "format" : "plain_text" },
       "submission_show_submitted" : 0,
       "submissions_view" : "default",
       "user_submissions_view" : "default",
       "form_status" : "ENTITYFORM_OPEN",
-      "roles" : { "1" : "1", "2" : "2", "30037204" : 0, "163670334" : 0 },
```
Comment (acrollet) [inline modules/roomify/roomify_contact/roomify_contact.features.inc:46]: @salvoscala let's use role names here instead of IDs. thanks! :)

**Review by acrollet — CHANGES_REQUESTED** [review]
I'd like to remove the role IDs before we release this.

**Comment by salvoscala** [comment]
@acrollet I think this is the way the form is exported so we've to change the entityform module to import role names. Any thoughts?

**Review by acrollet — APPROVED** [review]

**Comment by acrollet** [comment]
@salvoscala hmm I spent some time looking, but there's not a clear-cut way to take care of this. Given that we are only actually relying on the anon and authenticated roles, whose IDs should stay fixed, we can do without going down this rabbit hole. ;)

**Pull request merged**